cmd/snap/cmd_quota_test.go

```diff
@@ -272,6 +376,51 @@ current:
 	c.Check(s.Stdout(), check.Equals, fmt.Sprintf(outputTemplate, 500))
 }
 
+func (s *quotaSuite) TestGetCpuQuotaGroupSimple(c *check.C) {
+	restore := main.MockIsStdinTTY(true)
```

Comment:
(nitpick) do we actually need all of those or are they cargo-culted ;)

Reply:
Cargo-culted! And now gone!